### PR TITLE
Fix add/edit contentlet dialog issues

### PR DESCRIPTION
**Pull request by hmoreras**

### CODE REVIEW

File: src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts [hunk 1]
```diff
@@ -247,6 +247,7 @@ export class DotEditContentComponent implements OnInit {
             element it's available in the DOM
         */
         setTimeout(() => {
+            window.focus();
```
[the remark this answers is not in the document]
Comment (fmontes) [reply src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts]: what is this? :D

File: src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts [hunk 2]
```diff
@@ -247,6 +247,7 @@ export class DotEditContentComponent implements OnInit {
...
             const editContentletIframeEl = this.contentletActionsIframe.nativeElement;
 
             /*
```
[the remark this answers is not in the document]
Comment (fmontes) [reply src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts]: @freddyucv draggable feature is fixed here: ```[draggable]="false"``` but the other looks like is not.

[the remark this answers is not in the document]
Comment (freddyucv) [reply src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts]: I mean this: 

- Update the title message key, is broken in core

but humberto tell me that it is was broken in core

File: src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (fmontes) [reply src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts]: Ok we talked about this and this is not a solution, @hmoreras let me know if the solution we discuseed work.

File: src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts [hunk 3]
```diff
@@ -186,17 +187,36 @@ export class DotEditContentComponent implements OnInit, OnDestroy {
...
 
+    /**
+     * Bind events on "contentlet Actions" iFrame loaded in the dialog.
+     *
+     * @param $event
+     * @memberof DotEditContentComponent
+     */
+    onContentletActionLoaded($event: any): void {
+        const editContentletIframeEl = $event.target;
+        if (editContentletIframeEl.contentDocument.body.innerHTML !== '') {
```
[the remark this answers is not in the document]
Comment (hmoreras) [reply src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts]: yes the body html es empty.

File: src/app/portlets/dot-edit-page/content/dot-edit-content.component.spec.ts [hunk 4]
```diff
@@ -464,7 +463,15 @@ describe('DotEditContentComponent', () => {
             siteServiceMock.setFakeCurrentSite(mockSites[1]);
             expect(dotHttpErrorManagerService.handle).toHaveBeenCalledWith(fake500Response);
             expect(dotRouterService.gotoPortlet).toHaveBeenCalledWith('/c/site-browser');
+        });
+    });
 
+    describe('dialog configuration', () => {
+        it('should not be draggable, have dismissableMask and be a modal', () => {
+            const dialog = fixture.debugElement.query(By.css('p-dialog'));
+            expect(false).toEqual(dialog.nativeElement.draggable);
```
Comment (fmontes) [inline src/app/portlets/dot-edit-page/content/dot-edit-content.component.spec.ts]: ```expect(dialog.nativeElement.draggable).toEqual(false);```

First what we are asserting and then the value.

File: src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (freddyucv) [reply src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts]: I think we dont have any test to check this case

**Pull request merged**